cabot/cabotapp/tasks.py

```diff
@@ -96,7 +96,6 @@ def update_shifts():
 def reset_shifts(schedule_id):
     try:
         schedule = models.Schedule.objects.get(id=schedule_id)
-        models.delete_shifts(schedule)
```

Comment:
This was in case we switched schedules to something totally different, but clearing up non-updated entries serves the same purpose